CHANGES.rst

```diff
@@ -7,6 +7,19 @@ Features
 - Add support for Python 3.8.
   See https://github.com/Pylons/pyramid/pull/3547
 
+- New security APIs have been added to support a massive overhaul of the
+  authentication and authorization system. Read
+  "Upgrading Authentication/Authorization" in the "What's New in Pyramid 2.0"
```

Comment:
I understand instructions like this in Python warnings, but why not use a `ref` role here in a reST doc?

Reply:
the changes.rst is published to pypi and it does not understand most sphinx directives

Reply:
Right!  And you prefer giving instructions rather than risking link rot.